runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/manager/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Resource management classes for function execution.
```

Comment:
What resources? What management?

The things in here right now seems closer to being part of `control` than a separate package. What else will go in here?

Reply:
While the sdk harness manager will only expose a `RemoteBundle` to the operator, it is actually responsible for setting up and shutting down `RemoteEnvironment` and all `GrpcFnServer` instances according to their respective lifetimes. Because these classes span the other local packages, it made sense to me to start a new package for manager classes that aggregate "resources" from those various domains.

Reply:
Okay.  let me know what you think of it now. I've tried to pull out all of the constituent parts.